### ISSUE TITLE
fails if there are no parts

### ISSUE THREAD

**Issue by jbenet** [hendrikcech/multipart-stream#1]
``` js
> var mps = require('multipart-stream')
> var m = new Multipart('wellhellotherehowsitgoing')
> var m = new mps('wellhellotherehowsitgoing')
> m.pipe(process.stdout)
Error: SandwichStream no streams added error
  at Multipart.SandwichStream._read (/Users/jbenet/git/node-vinyl-to-multipart-stream/node_modules/multipart-stream/node_modules/sandwich-stream/lib/sandwich-stream.js:30:24)
  at Multipart.Readable.read (_stream_readable.js:324:10)
  at resume_ (_stream_readable.js:705:12)
  at doNTCallback2 (node.js:430:9)
  at process._tickDomainCallback (node.js:385:17)
```

I would expect the body to just be empty, not throw.

multipart libraries differ on what to do here. many http libs/servers allow a multipart in an HTTP request with no parts. docs differ in whether this is allowed.

**Comment by hendrikcech** [comment]
This module inherits from [connrs/node-sandwich-stream](https://github.com/connrs/node-sandwich-stream) so this has to fixed / changed downstream. The error is thrown on [line 30](https://github.com/connrs/node-sandwich-stream/blob/master/lib/sandwich-stream.js#L30).
Would you mind creating another issue there?

**Comment by jbenet** [comment]
done

**Comment by jbenet** [comment]
@hendrikcech sandwich-stream has been updated

**Comment by hendrikcech** [comment]
Great, thanks @connrs! I published v2.0.0 of this module to npm.
